ykrt/src/compile/j2/hir.y

```diff
@@ -59,8 +59,8 @@ Inst -> Result<AstInst, Box<dyn Error>>:
   | "LOCAL" ":" Ty "=" "CALL" "ID" "LOCAL" "(" Locals ")" {
       Ok(AstInst::Call { local: Some($1?.span()), ty: Some($3?), extern_: $6?.span(), tgt: $7?.span(), args: $9? })
     }
-  | "LOCAL" ":" Ty "=" "ABS" "LOCAL" {
-      Ok(AstInst::Abs { local: $1?.span(), ty: $3?, val: $6?.span() })
+  | "LOCAL" ":" Ty "=" "ABS" "LOCAL" IntMinPoison {
+      Ok(AstInst::Abs { local: $1?.span(), ty: $3?, val: $6?.span(), int_min_poison: $7? })
```

Comment:
I tried searching the GRM book for what `.span()` does, but couldn't find it. What does it do?

Reply:
https://docs.rs/lrpar/0.14.0/lrpar/trait.Lexeme.html#tymethod.span